### ISSUE TITLE
Add terminal linker explicitly or via contig

### ISSUE THREAD

**Issue by broomsday** [broomsday/monomerizer#4]
We want the N-term to ~link with the C-term in order to maximize the symmetry across the structure.  At least for `ProteinGenerator` where they will be sharing sequence logits.

There are two ways to accomplish this:

- [ ] copy the N-terminal residue of the first chain and put it at the C-terminus of the last chain.  This is garaunteed to work, but has the caveat that RoseTTAFold will adjust the coordinates of that overlapping residue, though as this should occur after diffusion it may not impact symmetry.
- [x] give the N-terminal residue in the contig twice, giving it again at the end.  This might not work, either `ProteinGenerator` may explicitly throw an error, or it may mess something up internally, but if it does function as intended it seems a more elegant solution.
